ClinGen/clincoded
# Connects to #991. Connects to #992. Extract PMID Add Modal to Add External Resource module

#### Connects to #992:

Pubmed modal code in GCI Curation Central has been extracted out to the Add External Resource module. Behavior has slightly changed to have confirmation step, like the variant modals.

*Testing:*

1. Go to GCI Curation Central
2. Confirm that adding PMID behaves as intended (including failure upon attempting to add two of the same PMID)

![image](https://cloud.githubusercontent.com/assets/4326866/18532407/bedb361e-7a90-11e6-8543-bda55926d2c4.png)

#### Connects to #991:

ClinVar modal has been updated to allow for only one of the same ClinVar ID to be added on the GCI Experimental/Family/Individual curation pages. Modal displays an appropriate error message when this is attempted.

*Testing:*

1. Go to GCI
2. Curate Experimental object w/ Variants
3. Add a variant
4. Attempt to add the exact same variant again, and confirm that an error message comes up
5. Edit a variant, and press 'Retrieve' without editing the value. This should still work.
6. Attempt to add another variant. This should work.
7. Repeat above w/ Family and Individual curation pages

![image](https://cloud.githubusercontent.com/assets/4326866/18532352/789b2ede-7a90-11e6-9a19-2e615e35efbd.png)
![image](https://cloud.githubusercontent.com/assets/4326866/18532361/8514e51a-7a90-11e6-931b-d3be4337c33c.png)
![image](https://cloud.githubusercontent.com/assets/4326866/18532381/9f892a5a-7a90-11e6-8069-14786e6e7fc2.png)
